### ISSUE TITLE
The app crashed when I tried to clear this app from notification panel after changing the song.

### ISSUE THREAD

**Issue by sagorahmed** [ZoidbergZero/Canora-MusicPlayer#7]
**Describe the bug**
When I tried to close the player from the notification panel after changing songs, then the app crashed.

**To Reproduce**
1. Open the app.
2. Play a song.
3. Minimize this app. (Make sure that song is running on background)
4. Swipe/pull down your phone's notification panel and change that running song.
5. Then clear the notification panel and notice the bug.

**Expected behavior**
The users should be able to clear/close the app from the notification panel without getting any trouble.

**Screenshots**
![bug](https://media.giphy.com/media/dWa2eLhXfTzlgTZPVD/giphy.gif)

**Informations:**
 - Device: Samsung a5
 - OS: Android 6.0.1
 - App Version: 0.3 

**LogCat**
```
05-21 13:48:28.086 30958 30958 E AndroidRuntime: FATAL EXCEPTION: main

05-21 13:48:28.086 30958 30958 E AndroidRuntime: Process: e.planet.musicman, PID: 30958

05-21 13:48:28.086 30958 30958 E AndroidRuntime: java.lang.IllegalStateException

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.media.MediaPlayer.isPlaying(Native Method)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at e.planet.musicman.MainActivity$1.onAnimationUpdate(MainActivity.java:309)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.animation.ValueAnimator.animateValue(ValueAnimator.java:1463)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.animation.ValueAnimator.animationFrame(ValueAnimator.java:1384)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.animation.ValueAnimator.doAnimationFrame(ValueAnimator.java:1427)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.animation.ValueAnimator$AnimationHandler.doAnimationFrame(ValueAnimator.java:759)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.animation.ValueAnimator$AnimationHandler$1.run(ValueAnimator.java:801)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:911)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.view.Choreographer.doCallbacks(Choreographer.java:686)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.view.Choreographer.doFrame(Choreographer.java:619)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:897)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:739)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:148)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7325)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)

05-21 13:48:28.086 30958 30958 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
```

**Comment by ZoidbergZero** [comment]
Fixed in 00a6309
I released v0.4 where this fix is implemented, Closing the Issue now.

**Comment by sagorahmed** [comment]
Hi @ZoidbergZero 
This issue is not occurring anymore in v0.4. 
Thanks for fixing this issue in a very short time, Amazing job.
